react/package.json

```diff
@@ -72,7 +72,26 @@
     "react-intl": "^2.4.0",
     "react-testing-library": "^5.6.0",
     "regenerator-runtime": "^0.13.1",
-    "ts-jest": "^23.10.5",
+    "ts-jest": "^24.0.1",
     "typescript": "3.5.2"
+  },
+  "jest": {
```

Comment:
Why not use https://github.com/vtex/test-tools? And follow the TS steps https://github.com/vtex/test-tools/tree/master/examples/typescript

I feel attacked

Reply:
Nice.

I didn't know about this test framework. I was just adjusting the existing test configuration.